src/styles/main.scss

```diff
@@ -0,0 +1,52 @@
+@import "reset";
+@import "variables";
+@import "animation";
+@import "mixins";
+
+body {
+  position: relative;
+  height: 100vh;
+  background-color: $backGroundColor;
+}
+
+.clock {
+  position: relative;
+  display: block;
+  background-color: $colorBgClock;
+  width: $clockSize;
+  height: $clockSize;
+  transform: translate(-100%, -50%);
+  left: 50%;
+  top: 50%;
```

Comment:
if you use calc(50% - size / 2), you can remove transform and will have centered element
(size - width or height depending on what you try to center: left or top)

Reply:
changed to **calc(50% - #{$clockSize} / 2);**